packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -136,8 +136,7 @@ export class VaCheckbox {
           onChange={this.handleChange}
         />
         <label htmlFor="checkbox-element">
-          {label}
-          {required && <span class="required">(*Required)</span>}
+          <span>{label} {required && <span class="required">(*Required)</span>}</span>
```

Comment:
This was necessary to align the label along the right edge of the checkbox instead of the text wrapping as described [here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/991#issuecomment-1240758320). Flexbox is being used on the `label` element to solve for that and in doing so, each of the elements  display as columns (3 columns originally):

<img width="446" alt="Screen Shot 2022-09-12 at 3 01 11 PM" src="https://user-images.githubusercontent.com/872479/189746262-32eec96c-6f14-4d73-b165-ad66786e18cd.png">

Wrapping the`{label}` and "required" together with a span made sure that the "required" text doesn't go into its own column and positions awkwardly to the right (like shown above).

<img width="360" alt="Screen Shot 2022-09-12 at 3 01 47 PM" src="https://user-images.githubusercontent.com/872479/189746558-f13d0b6b-105a-46a5-b43c-0aebed17bb6d.png">

Reply:
Since these changes aren't strictly related to the privacy agreement component, could we move the wrapping fixes to their own PR? I think that will help with release notes and searches in the future.

Reply:
@bkjohnson I suppose I could but I did document this update [in the development ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/991#issuecomment-1240758320) and ultimately decided to add it here so that it could pass design review.

Reply:
I don't think that having a separate PR would result in a change to the design review - the fix PR could be merged into `main` and we could pull it into this one.

Here's an example of why I think that could be good. If we look at the git history for a component we will see all the commits to it:

![Git history of the table web component](https://user-images.githubusercontent.com/2008881/189980719-62a49388-ed75-4fe9-83ba-e26552f3f426.png)

Having a fix to an existing component in the same PR (and eventually squashed commit) where we add a new component muddies things a bit. If we were to look at the history for `va-checkbox` we wouldn't see a commit like "Fix styles when label text gets wrapped" - we would just see "Privacy Agreement Web Component" and would have to dig a bit deeper to realize that there was more going on.

That will also make the git blame more specific. If I look at the blame while on this PR branch, it gives me a good idea that the span was added for a label thing:

![Git blame for the PR branch. The line with the span is highlighted and the blame reads "Fix va-checkbox label allignment"](https://user-images.githubusercontent.com/2008881/189981783-675f095a-7863-4757-b905-4cd6ef039026.png)

If we were to merge this PR as-is, then the squash would result in the blame showing the commit message for adding the privacy agreement web component, so again we would have to dig a bit deeper to understand the real reason behind the change. And since this fix is for a visual bug that only happens in very specific situations, if someone were to remove this wrapping span in the future as part of a refactor for simplifying the markup or something like that, they wouldn't know that they would be causing a regression unless they happened to be testing with a really long label or happened to dig deep enough to realize that the PR that added the privacy agreement component was also fixing a bug in the checkbox component.

I think it's ok to link the PRs together and make it clear that one resulted from a bug during work on the other, but having a more specific git history seems like the kind of thing that could make our jobs easier in the future when we're looking back to get an idea of what was done and why.

Reply:
👍  That commit is reverted and a separate PR is up: https://github.com/department-of-veterans-affairs/component-library/pull/530